libbpf-tools/mountsnoop.c

```diff
@@ -35,9 +37,14 @@
 
 static volatile sig_atomic_t exiting = 0;
 
-static pid_t target_pid = 0;
-static bool emit_timestamp = false;
-static bool output_vertically = false;
+static struct env {
+	pid_t target_pid;
+	bool emit_timestamp;
+	bool output_vertically;
+	const char *mntnsmap;
+	const char *containersmap;
+} env;
```

Comment:
We're trying to keep the changes to upstream as small as possible. I'd suggest to avoid introducing the env structure and  add three more variables there instead.